06-webpack/classworks/Chernov_Artyom/task2/webpack.config.js

```diff
@@ -0,0 +1,64 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const {CleanWebpackPlugin} = require('clean-webpack-plugin');
+const { DefinePlugin } = require ('webpack');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
+
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    main: './src/index.js',
+  },
+  optimization: {
+    minimize: true,
+    minimizer: [new TerserPlugin({}), new OptimizeCSSAssetsPlugin({})],
+    splitChunks: {
```

Comment:
You don't need to use splitChunks in this app cause we have single entry point and also don't have chunks